flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js

```diff
@@ -131,23 +131,29 @@ class A32NX_Boarding {
 
         // Sound Controllers
         if ((currentPax < paxTarget) && boardingStartedByUser == true) {
-            await SimVar.SetSimVarValue(`L:A32NX_PAX_BOARDING_SOUND`, "Bool", true);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_BOARDING", "Bool", true);
             this.isBoarding = true;
         } else {
-            await SimVar.SetSimVarValue(`L:A32NX_PAX_BOARDING_SOUND`, "Bool", false);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_BOARDING", "Bool", false);
         }
 
         if ((currentPax > paxTarget) && boardingStartedByUser == true) {
-            await SimVar.SetSimVarValue(`L:A32NX_PAX_DEBOARDING_SOUND`, "Bool", true);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_DEBOARDING", "Bool", true);
         } else {
-            await SimVar.SetSimVarValue(`L:A32NX_PAX_DEBOARDING_SOUND`, "Bool", false);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_DEBOARDING", "Bool", false);
         }
```

Comment:
```suggestion
        await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_DEBOARDING", "Bool", currentPax > paxTarget && boardingStartedByUser == true);        
```

Reply:
learnt something new, thanks for the suggestions, implemented 🎉